M2/Macaulay2/packages/SagbiGbDetection.m2

```diff
@@ -1,6 +1,6 @@
 newPackage(
         "SagbiGbDetection",
-        Headline => "this package verifies whether the given generators of an ideal form the Gr\"obner basis or whether the given generators of a finitely generated subalgebra of a polynomial ring are the SAGBI basis",
+        Headline => "finding term orders for which the given generators of an ideal/algebra form a Gröbner/SAGBI basis",
```

Comment:
I'm trying to make all package headlines uniform, which includes making headlines less than 100 characters long. @timduff35 is this change okay with you?